### ISSUE TITLE
[Accessibility]Combo box menu option position not Narrate by JAWS for IE11

### ISSUE THREAD

**Issue by MdAmeerMind** [OfficeDev/office-ui-fabric-react#8294]
**Bug Report**

- Package version(s): Current examples at https://developer.microsoft.com/en-us/fabric#/components/ComboBox#Variants
- Browser and OS versions: IE 11 + JAWS 2019

**Actual Behaviour**
When user navigate to Combo box menu option, JAWS only narrating menu option text, menu option position not narrated. 

**Expected:**
JAWS should provide position of menu item along with menu text. 

**Observation**
Menu item position narrate by Narrator for 

- Chrome+NVDA 
- Edge+ JAWS 2019

**Comment by KevinTCoughlin** [comment]
@MdAmeerMind thanks for your report. We focus on Edge + Narrator compatibility which seems to be working as expected in the above scenarios (see output below).

```shell
Single-select ComboBox (uncontrolled, allowFreeform: T, autoComplete: T), Option G, 11 of 11, selected, 
10 of 11, non-selected, Option J
9 of 11, non-selected, Option I
8 of 11, non-selected, Option H
```

Since it works in Narrator, we are likely providing the correct attributes and this is more likely an issue with NVDA / JAWS integration and should be raised in their bug report channels.

If there are incorrect / missing ARIA attributes that would offer this compatibility please let us know and we will work to add them.

**Comment by MdAmeerMind** [comment]
Hi @KevinTCoughlin, thank u for clarification. 
Attribute require to narrate list option position are already there. Even observe same behaviour on reference site for ARIA implementation . [https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html)
It seems it is JAWS issue with IE 11.

**Comment by KevinTCoughlin** [comment]
Thanks for following up @MdAmeerMind. I believe the JAWS open bug tracker can be found at https://github.com/FreedomScientific/VFO-standards-support/issues. I recommend opening an issue for the above there. Feel free to cross-reference this issue.

I'm going to let the bot close this after some time unless there is action the Fabric team needs to take here.

**Comment by MdAmeerMind** [comment]
Hi @KevinTCoughlin ,
Same issue was already raised on JAWS support .[ https://github.com/FreedomScientific/VFO-standards-support/issues/53]( https://github.com/FreedomScientific/VFO-standards-support/issues/53)
They provided solution for issue.
I had tried there suggestion by explicitly set the [aria-setsize](https://www.w3.org/TR/wai-aria-1.1/#aria-setsize) and [aria-posinset](https://www.w3.org/TR/wai-aria-1.1/#aria-posinset) attributes and it works for IE.
Here are[ link](https://www.w3schools.com/code/tryit.asp?filename=G24I7Z8PVRSS) of code changes I have made for demo.

**Comment by KevinTCoughlin** [comment]
> Hi @KevinTCoughlin ,
> Same issue was already raised on JAWS support .[ https://github.com/FreedomScientific/VFO-standards-support/issues/53](https://github.com/FreedomScientific/VFO-standards-support/issues/53)
> They provided solution for issue.
> I had tried there suggestion by explicitly set the [aria-setsize](https://www.w3.org/TR/wai-aria-1.1/#aria-setsize) and [aria-posinset](https://www.w3.org/TR/wai-aria-1.1/#aria-posinset) attributes and it works on IE.
> Here are[ link](https://www.w3schools.com/code/tryit.asp?filename=G24I7Z8PVRSS) of code changes I have made for demo.

@MdAmeerMind, are you able to make the changes in your application to satisfy JAWS? Or are you still blocked awaiting changes within `office-ui-fabric-react`?

**Comment by MdAmeerMind** [comment]
Hi @KevinTCoughlin ,
I am blocked awaiting changes within office-ui-fabric-react component. As combo box is render as input and div's, I am unable to make changes in that logic. so better if changes made from office-fabric team.

**Comment by KevinTCoughlin** [comment]
I'll take a look at JAWS' proposed workaround and see if it can be applied to OUFR either at the library level or app level. Unsure how it will go since its a confirmed JAWS bug as mentioned above.

**Comment by jspurlin** [comment]
@dzearing This is same underlying issue as #8930 

This should be WF and at most JAWS could be made aware of the bug they have
